cmd/juju/application/bundle.go

```diff
@@ -1390,6 +1390,18 @@ func buildModelRepresentation(
 			return nil, errors.Errorf("unexpected tag kind for annotations: %q", kind)
 		}
 	}
+	// Add in the model sequences.
+	sequences, err := apiRoot.Sequences()
```

Comment:
Nit - preexisting code, but apiRoot typically is used to refer to the base api connection from which we then do a <api-name>.NewClient(apiRoot). Perhaps rename to modelConfigAPI which would be consistent with other commands.

Reply:
The apiRoot here is an adapter that wraps many api clients. It is kinda icky IMO.